safe_transaction_service/tokens/clients/coingecko_client.py

```diff
@@ -36,8 +36,10 @@ class CoingeckoClient:
     }
     base_url = "https://api.coingecko.com/"
 
-    def __init__(self, network: Optional[EthereumNetwork] = None):
-        self.http_session = requests.Session()
+    def __init__(
+        self, network: Optional[EthereumNetwork] = None, request_timeout: int = 10
```

Comment:
Maybe should be `request_timeout Optional[int] = 10`, the same in other places

Reply:
Is not optional as it's always expecting an int